events/messageCreate.js

```diff
@@ -2,7 +2,6 @@
 // Note that due to the binding of client to every event, every event
 // goes `client, other, args` when this function is run.
 
-const Event = require("../base/Event");
```

Comment:
I think this was a mistake, I don't think you wanted to actually remove it.

Reply:
Uhhh, you're correct. Will fix, no idea why it was deleted.

Reply:
Fixed in latest commit.